src/bidiMapper/domains/script/ScriptProcessor.ts

```diff
@@ -50,6 +51,21 @@ export class ScriptProcessor {
   async addPreloadScript(
     params: Script.AddPreloadScriptParameters
   ): Promise<Script.AddPreloadScriptResult> {
+    if (params.contexts) {
+      if (params.contexts.length === 0) {
+        throw new InvalidArgumentException('Invalid empty contexts list.');
```

Comment:
How about: "Contexts list is empty".

Invalid is implicit in the error type, we don't need to repeat it in the error message